test/ranking.jl

```diff
@@ -37,3 +37,76 @@ s = ["c", "a", "b", "d", "d", "b", "e", "d"] # s is a vector of strings ordered
 @test tiedrank(s) == tiedrank(x)
 @test tiedrank(x, rev = true) == tiedrank(-x)
 @test tiedrank(x, lt = (x, y) -> isless(y, x)) == tiedrank(-x)
+
+@testset "quantilerank and percentrank" begin
+    @testset "value as number and array" begin
+        @testset ":inc and :exc" begin
+            v1 = [1, 1, 1, 2, 3, 4, 8, 11, 12, 13]
+            v2 = [1, 2, 3, 6, 6, 6, 7, 8, 9]
+            v3 = [1, 2, 4, 3, 4]
+            v4 = [1, 2, 1, 3, 4]
+            @test quantilerank(v1, 2, method=:inc) == 1/3
+            @test quantilerank(v1, 4, method=:inc) == 5/9
+            @test quantilerank(v1, 8, method=:inc) == 2/3
+            @test quantilerank(v1, 5, method=:inc) == 7/12
+            @test percentrank(v1,  2, method=:inc) == 100 * (1/3)
```

Comment:
The tests for `percentrank` are not completely systematic. Given how simple the definition is, I'd remove all tests for it, and instead have a single place where you just check that `percentrank` gives the expected result when compared with `quantilerank`, both without and with `method` specified.

Reply:
I got you. So, I will :

- Remove all the percentilerank tests, lefting only the quantilerank tests of unique cases.
- Add percentilerank tests that compare if percentilerank results is equal quantilerank result times 100. 

Question: 
Do you have suggestion on what cases I test percentilerank? 
because I think the idea it's also to minimize the number of redundant tests.

Reply:
In commit 3e30dd1, I separated the tests into quantilerank and percentilerank sections, by removing the percentilerank tests and putting it in the new section.

The code got more lines and, in my opinion, I preferred to keep it as it was before. 
Maybe we could reduce the cases (because I tried to replicate all the examples in docs, so there is no doubt). 
Anyway, I'll go with what you prefer and suggest me =)

Reply:
Sorry, when I said "both without and with `method` specified", I meant that only two tests are needed. We don't have to cover all cases as they are already tested via `quantilerank`.

Reply:
Now, I got what you're meaned.

In the next commit, I'll include the two tests cases of percentilerank in the `:inc` and `:exc` section of quantilerank tests. This will reduce the amount of lines and the number of tests. 

Since I put all the examples founded in the docs, this could a bit too much. So if you want to reduce some tests, by deleting some lines, I see no problem in that.

Reply:
Done in commit b74e8dc.